model/src/wav_import_export.F90

```diff
@@ -145,6 +145,9 @@ subroutine advertise_fields(importState, ExportState, flds_scalar_name, rc)
       call fldlist_add(fldsFrWav_num, fldsFrWav, 'Sw_vstokes')
     else
       call fldlist_add(fldsFrWav_num, fldsFrWav, 'Sw_z0')
+      call fldlist_add(fldsFrWav_num, fldsFrWav, 'Sw_wavsuu')
```

Comment:
@uturuncoglu should this be in the else for the cesm coupled? or should this be completely outside of this if/else statement?

Reply:
@JessicaMeixner-NOAA I think those variables are not used by CESM but we are using under UFS Coastal. So, I did not want to change anything in CESM side by putting them outside of the if statement.